prog-o-meter.py

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
-The Prog-o-meter.py program is for tracking progress during a #100DaysofCode challenge (or any other #100DaysofX challenge).
+The Prog-o-meter.py program is for tracking progress during a
+100DaysofCode challenge (or any other #100DaysofX challenge).
 
-The program gives a graphic overview of ones progress through the challenge, by showing a bar containing 100 fields, showing completed days as colored in, and remaining days as white
-Currently, the prog-o-meter only tracks your progress in days, but there are plans for many new features, so stay tuned.
-The prog-o-meter is developed as an opensource project, and any contributions are welcome, at https://github.com/lineaba/prog-o-meter
+The program gives a graphic overview of ones progress through the
+challenge, by showing a bar containing 100 fields, showing completed
+days as colored in, and remaining days as white Currently, the

```

Comment:
Good catch!  I might have deleted that when I was merging the lines.  Often times merging the lines got really, really weird.

Reply:
Haha, no it was definetely me who never put it in. I looked in the original code. There was never a period, just didn't look as weird because there was a line break.